tensorflow/contrib/tensorrt/resources/trt_allocator.cc

```diff
@@ -27,12 +27,12 @@ namespace tensorflow {
 namespace tensorrt {
 
 // std::align is not supported, so this method mimic its behavior.
-void* Align(size_t alignment, size_t size, void*& ptr, size_t& space) {
-  QCHECK_GT(alignment, 0) << "alignment must be greater than 0.";
+void* Align(uint64_t alignment, uint64_t size, void*& ptr, uint64_t& space) {
```

Comment:
I think keeping alignment at 64 bit is overkill. Even 32bit is too much

Reply:
Thanks for the suggestions, I added comment for that, also add back the tensor name (conditionally).
Would you help to approve the PR again? Thanks!